pudb/debugger.py

```diff
@@ -214,6 +223,32 @@ def __init__(self, stdin=None, stdout=None, term_size=None, steal_output=False,
         # Okay, now we have a debugger
         self._current_debugger.append(self)
```

Comment:
Could we get rid of `_current_debugger` in favor of the singleton mechanism?

Reply:
That is logic in place for `_get_debugger`. This has to remain. That is all fine.

Trying to instantiate a debugger again (and thus previously hitting the ValueError) is what pytest is doing. Thus my code only works around that specific pytest problem. All logic around `_get_debugger` is fine, but pytest is just not calling it.